src/userobjects/MoabSkinner.C

```diff
@@ -6,6 +6,9 @@
 #include "BinUtility.h"
 #include "GeometryUtils.h"
 #include "UserErrorChecking.h"
+#include "DisplacedProblem.h"
+#include "FEProblemBase.h"
+#include "MooseMesh.h"
```

Comment:
I think if you move this line into the header, you can kill two birds with one stone and eliminate the `class MooseMesh` thing.